TarkovMonitor/MainBlazorUI.cs

```diff
@@ -92,6 +92,14 @@ public MainBlazorUI()
             blazorWebView1.RootComponents.Add<TarkovMonitor.Blazor.App>("#app");
 
             blazorWebView1.WebView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;
+
+            if (Properties.Settings.Default.minimizeAtStartup)
+            {
+
+                this.WindowState = FormWindowState.Minimized;
+                this.ShowInTaskbar = false;
```

Comment:
Unless there is a reason I cannot see, You do not need to call line 100 or 101. When you set the WindowState, that event will trigger the MainBlazorUI_Resize function which will hide and set the notification icon.

Reply:
That was my assumption as well, but it seemed that the handler was not called for changes made in the constructor. Unless I've done something wrong?

Reply:
I checked out your PR and tested without those lines and it still works as expected. Do you want to double-check on your end?

Reply:
Will do. I'm on vacation so it might be a day or two. I'll try to fit it in tomorrow though.

Reply:
I'm a bit late to the party here, but has this been sorted?

Reply:
Woah. I completely forgot to check back on this. I've just been running a build I made with the changes :)

I'll confirm in a couple of hours

Reply:
Great, thanks! Apologies for neglecting this PR; been stretched in a lot of different directions the past few months.

Reply:
I went ahead and checked just now. Without `this.ShowInTaskbar = false;` the icon still appears on the taskbar, but the window is not rendered, as you can see in the hover preview:
![image](https://github.com/the-hideout/TarkovMonitor/assets/16705594/5fbaa692-f99d-4efc-8bfc-18547793ded2)
Restoring the window and then re-minimizing it will return it to its typical behavior.

It seems `notifyIconTarkovMonitor.Visible = true;` is not needed, however.

In addition, I've noticed an issue where when the app is launched in a minimized state, the unrendered preview window is visible in the Windows task switcher until the app is first interacted with:
![image](https://github.com/the-hideout/TarkovMonitor/assets/16705594/413cfd05-850a-48e0-97a7-2901ec219254)


This was tested using the `publish` task that VS Code automatically picked up.  
Perhaps that is not the correct way to build it and that's why I'm seeing a different result?  
My tasks config is as follows:
```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "build",
      "command": "dotnet",
      "type": "process",
      "args": [
        "build",
        "${workspaceFolder}/TarkovMonitor/TarkovMonitor.csproj",
        "/property:GenerateFullPaths=true",
        "/consoleloggerparameters:NoSummary"
      ],
      "problemMatcher": "$msCompile"
    },
    {
      "label": "publish",
      "command": "dotnet",
      "type": "process",
      "args": [
        "publish",
        "${workspaceFolder}/TarkovMonitor/TarkovMonitor.csproj",
        "/property:GenerateFullPaths=true",
        "/consoleloggerparameters:NoSummary"
      ],
      "problemMatcher": "$msCompile"
    },
    {
      "label": "watch",
      "command": "dotnet",
      "type": "process",
      "args": [
        "watch",
        "run",
        "--project",
        "${workspaceFolder}/TarkovMonitor/TarkovMonitor.csproj"
      ],
      "problemMatcher": "$msCompile"
    }
  ]
}
```

Reply:
Moving the logic to `OnShown` resolves all of the issues above and reuses `MainBlazorUI_Resize`. I've updated the PR

```cs
protected override void OnShown(EventArgs e)
{
    base.OnShown(e);

    if (Properties.Settings.Default.minimizeAtStartup)
    {

        WindowState = FormWindowState.Minimized;
    }
}
```